framework/lib/hooks/use-select-callbacks/use-select-callbacks.ts

```diff
@@ -14,9 +14,9 @@ export const useSelectCallbacks = <T extends BasicOption>({
 
   const handleChange = (val: MultiValue<T> | SingleValue<T>, event: ActionMeta<T>) => {
     onChange(val, event)
-    if (!isMulti && !isNil(value)) {
+    if (!isMulti && !isNil(val)) {
       onAdd((val as T).value)
-    } else if ((val as T[]).length > (items.current as T[]).length) {
+    } else if (val && ((val as T[]).length > (items.current as T[]).length)) {
```

Comment:
!isNil(val) here as well?

Reply:
yes, isNil should be used here as well for consistency

Reply:
Fixed :)